spec/integrations/fixtures/rspec/before_all_isolator_fixture.rb

```diff
@@ -38,20 +38,22 @@ class User < ActiveRecord::Base
 end
 
 describe "before_all + Isolator", :transactional do
-  before_all do
-    @user = User.create
-    SampleJob.perform_async(true)
-    @user.commited = true
-  end
+  context "with an implicit transaction" do
```

Comment:
Not 100% certain if the intention for this test was to demonstrate that Isolator blows up **after** the example itself.